internal/server/data/migrations_test.go

```diff
@@ -838,7 +861,8 @@ func dumpSchema(t *testing.T, conn string) string {
 
 	out := new(bytes.Buffer)
 	// https://www.postgresql.org/docs/current/app-pgdump.html
-	cmd := exec.Command("pg_dump", "--no-owner", "--no-tablespaces", "--schema-only", "--schema=testing")
+	args = append([]string{"--no-owner", "--no-tablespaces", "--schema=testing"}, args...)
```

Comment:
Wouldn't this be neater if the static options come after?

```suggestion
	args = append(args, "--no-owner", "--no-tablespaces", "--schema=testing")
```

Reply:
Ya, that should work as well. I was concerned about modifying the backing array of `args`, but since this variable comes from a varargs, I don't know if that's really a problem.

The one difference is that I think by putting `args` at the end the caller can decide to override some of the pre-defined args, but maybe we won't need that. 

I think it should be safe to move `args` to the first argument.